### PR TITLE
Add Domain: e-giftvip.com

### PR DESCRIPTION
**Pull request by actually-akac**
## Domain/URL/IP(s) where you have found the Phishing:
https://e-giftvip.com

## Related external source
https://urlscan.io/result/4163b4f3-3e70-4994-8bee-17cc4f3058e5

## Describe the issue
This is running a phishing attack against **Discord**. The website displays a fake login form to gather Discord account credentials.

### Screenshot
urlscan

### CODE REVIEW

File: add-domain [hunk 1]
```diff
@@ -1039,6 +1043,3 @@ youareanidiot.cc
 youthful-engelbart.34-125-197-109.plesk.page
 zc11m.csb.app
 zttmct-tlemp.web.app
-disocrds.gift
-steamcommunitvs.com
-disceord.gift
```
Comment (spirillen) [inline add-domain:1044]: These three domains needs to stay, they are potential active

image

File: add-domain [hunk 2]
```diff
@@ -611,7 +613,6 @@ helpbusinessportal.ml
 helpcenter-verify.com
 helpcontactservicebusiness.com
 helpformediaportal.com
-help-for-ukraine.eu
```
Comment (spirillen) [inline add-domain:614]: Please restore as potential active

image

File: add-domain [hunk 3]
```diff
@@ -1,5 +1,3 @@
-wsetransport.com
-me2.do
```
Comment (spirillen) [inline add-domain:2]: Please restore as potential threats

image

**Review by spirillen — CHANGES_REQUESTED** [review]

**Comment by actually-akac** [comment]
@spirillen I sorted the file so that it follows the recommendations. Do I need to revert it for it to be merged?

**Comment by spirillen** [comment]
> @spirillen I sorted the file so that it follows the recommendations. Do I need to revert it for it to be merged?

Not at all, I was simply me who haven't noticed it :) there are only the last one which needs a [@rebase](https://mypdns.org/rebase)

**Pull request merged**